lib/box-client.js

```diff
@@ -374,6 +386,63 @@ BoxClient.prototype.collaborationRoles = {
 };
 
 /**
+* Enum of valid webhooks event triggers
+*
+* @readonly
+* @enum {webhookTriggerType}
+*/
+BoxClient.prototype.webhookTriggerType = {
+	FILE: {
+		UPLOADED: 'FILE.UPLOADED',
+		PREVIEWED: 'FILE.PREVIEWED',
+		DOWNLOADED: 'FILE.DOWNLOADED',
+		TRASHED: 'FILE.TRASHED',
+		DELETED: 'FILE.DELETED',
+		RESTORED: 'FILE.RESTORED',
+		COPIED: 'FILE.COPIED',
+		MOVED: 'FILE.MOVED',
+		LOCKED: 'FILE.LOCKED',
+		UNLOCKED: 'FILE.UNLOCKED'
+	},
+	COMMENT: {
+		CREATED: 'COMMENT.CREATED',
+		UPDATED: 'COMMENT.UPDATED',
+		DELETED: 'COMMENT.DELETED'
+	},
+	TASK_ASSIGNMENT: {
+		CREATED: 'TASK_ASSIGNMENT.CREATED',
+		UPDATED: 'TASK_ASSIGNMENT.UPDATED'
+	},
+	METADATA_INSTANCE: {
+		CREATED: 'METADATA_INSTANCE.CREATED',
+		UPDATED: 'METADATA_INSTANCE.UPDATED',
+		DELETED: 'METADATA_INSTANCE.DELETED'
+	},
+	FOLDER: {
+		CREATED: 'FOLDER.CREATED',
+		DOWNLOADED: 'FOLDER.DOWNLOADED',
+		RESTORED: 'FOLDER.RESTORED',
+		DELETED: 'FOLDER.DELETED',
+		COPIED: 'FOLDER.COPIED',
+		MOVED: 'FOLDER.MOVED'
+	},
+	WEBHOOK: {
+		DELETED: 'WEBHOOK.DELETED'
+	}
+};
+
+/**
+* Enum of Box file or folder object type
+*
+* @readonly
+* @enum {targetType}
```

Comment:
I'm not sure that this is the right name for this enum type.  Consider using `ItemType` or `ObjectType`, since this is on BoxClient it seems a bit more general